Update plato to 0.9.34

LinusCDE opened the pull request:
Finally got to updating it.

A bunch of coding structures changed in plato but should build and run as usual.

https://github.com/LinusCDE/plato/releases/tag/0.9.34-rm-release-13

LinusCDE committed:
Update plato to 0.9.34

LinusCDE committed:
Bugfix

LinusCDE commented:
Well, seems rust:v2.3 is not recent enough again

https://github.com/toltec-dev/toltec/actions/runs/4367424650/jobs/7638731380:
```
[   ERROR] toltec.builder: plato [rmall]: error: package `time v0.3.20` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.61.0-nightly
```

LinusCDE commented:
Should build once https://github.com/toltec-dev/toolchain/pull/14 get's merged and released with a new tag (e.g. v2.3.1 or v2.4). Just need to update image to that then here and it should work. :crossed_fingers:

ju6ge commented:
OMG thank you so much for updating this. I was thinking about doing it myself. I am developing a plato extenstion in order to be able to sync zotero papers using plato. Would be great to be able to easly have papers availible on the remarkable. I was noticing a lot differences in how newer plato version are doing the communication with the extensions. Hopefully this will be available soon so that I don't have an excuse anymore and finish the extension.

LinusCDE committed:
Empty commit to trigger build

Eeems committed:
Use rust:v2.3.1

Eeems committed:
Merge branch 'testing' into feature/update-plato

LinusCDE committed:
Default to building armv7

Will be fixed in next commit:
https://github.com/LinusCDE/plato/commit/fbfaf3d7b4eb22205ad819a3dc12e4eb999cda15

LinusCDE committed:
Add new build-dependency: patchelf

LinusCDE committed:
Fix syntax error
